Makefile

```diff
@@ -15,14 +16,15 @@ help:
 	@echo "stop              Stops the Docker container."
 	@echo ""
 	@echo "Variables:"
-	@echo "SIMFILE_PATH       Path to the sim.json file."
+	@echo "SIMFILE_PATH      Path to the sim.json file."
 	@echo "LOG_LEVEL         Set the logging level (default: info) e.g. <make run LOG_LEVEL=debug>"
 	@echo "HELP              Set to true to print the help message (default: false) e.g. <make run HELP=true>"
 	@echo "PRINT_BATCH_SIZE  Set the batch size for printing the results e.g. <make run PRINT_BATCH_SIZE=100>"
 	@echo "TOTAL_TIME        Set the total time for the simulation e.g. <make run TOTAL_TIME=1000>"
+	@echo "DATA_DIR          Set the data directory for the simulation containing simulation files and results e.g. <make run-docker DATA_DIR="/Users/anon/data>""
 
 run-docker:
-	docker run -d --rm --name sim-ln --init -v simln-data:/data -e SIMFILE_PATH=/data/sim.json -e LOG_LEVEL=$(LOG_LEVEL) -e HELP=${HELP} -e PRINT_BATCH_SIZE=${PRINT_BATCH_SIZE} -e TOTAL_TIME=${TOTAL_TIME} sim-ln
+	docker run -d --rm --name sim-ln --init -v simln-data:/data -e SIMFILE_PATH=/data/sim.json -e DATA_DIR=${DATA_DIR} -e LOG_LEVEL=$(LOG_LEVEL) -e HELP=${HELP} -e PRINT_BATCH_SIZE=${PRINT_BATCH_SIZE} -e TOTAL_TIME=${TOTAL_TIME} sim-ln
```

Comment:
`-e SIMFILE_PATH=/data/sim.json` should be `-e SIMFILE_PATH=${SIMFILE_PATH}` now right? Given the parameter is not required anymore plus we want the user to be able to re-define the location if desired

Reply:
Gotcha. fixed.